app/views/admin/merchants/index.html.erb

```diff
@@ -1,3 +1,19 @@
-<% @merchants.each do |merchant| %>
-  <p><%= link_to merchant.name, admin_merchant_path(merchant) %></p>
-<% end %>
\ No newline at end of file
+<div id="enabled-merchants">
+  <h2>Enabled Merchants</h2>
+  <% @merchants.all_enabled.each do |merchant| %>
+    <div class="merchant-<%= merchant.id %>">
+      <p><%= link_to merchant.name, admin_merchant_path(merchant) %></p>
+      <%= button_to "Disable", admin_merchant_path(id: "#{merchant.id}", status: "1"), method: :patch %>
```

Comment:
So if I am looking at this correctly you are passing in the params status back to the controller then you can use that in the if statement? I really like this and will look into button_to more!

Reply:
yup, exactly! it's passing in a param (status) with a value of "1" (disabled) and which is then used in the update method in the admin::merchant controller to update the status! pretty neat!

Reply:
button_to!!!!